news/colorupdate.rst

```diff
@@ -4,7 +4,6 @@
 
 **Changed:**
 
-* Change of two colors for higher contrast: ``#AB987A`` to ``#918770`` and ``#984B43`` to ``#6C5050``
 * Capitalized the hex color codes
```

Comment:
I think this should be under fixed.  Changed refers to changes that would affect the user.

Reply:
> I think this should be under fixed. Changed refers to changes that would affect the user.

Noted. Moved and please review.